### PR TITLE
feat: Add option to draw shadow behind toasts

### PR DESCRIPTION
**Pull request by woelper**
This PR exposes the option to use a shadow on notifications in order to improve visibility. The shadow is optional.

### CODE REVIEW

**Comment by bircni** [comment]
Maybe also extend the examples to show the shadow

**Comment by woelper** [comment]
> Maybe also extend the examples to show the shadow

I added a button to add shadows. Hope this works for you!

**Comment by bircni** [comment]
Could not spot any difference
with shadow:
image

without shadow:
image

**Comment by bircni** [comment]
I suggest changing the examples like this:
[all.rs.txt](https://github.com/user-attachments/files/17163748/all.rs.txt)

**Comment by woelper** [comment]
> I suggest changing the examples like this: [all.rs.txt](https://github.com/user-attachments/files/17163748/all.rs.txt)

Done!

File: src/lib.rs [hunk 1]
```diff
@@ -159,6 +162,12 @@ impl Toasts {
         self
     }
 
+    /// Margin or distance from screen to toasts' bounding boxes
```
Comment (ItsEthra) [inline src/lib.rs:165]: This comment looks confusing, how is it related to shadows?

Comment (woelper) [reply src/lib.rs]: I missed this one! Fixed!

**Comment by ItsEthra** [comment]
Also it says there are conflicts? you may need to rebase. and could you add a line to changelog I just created.

**Comment by woelper** [comment]
I have no idea how to resolve the conflicts in the fork to be honest... what I see from the `gh` tool and the web interface is not really helpful

**Comment by woelper** [comment]
I think I got it done using the web editor.

**Review by ItsEthra — APPROVED** [review]
Thanks

**Pull request merged**